src/main/java/net/smartcosmos/dao/relationships/RelationshipDao.java

```diff
@@ -28,6 +33,19 @@
     Optional<RelationshipResponse> findByUrn(String accountUrn, String urn);
 
     /**
+     * Finds a relationship matching specified entity, related entity and type.
+     *
+     * @param accountUrn
+     * @param relationshipLookupSpecific
+     * @return
+     * @throws ConstraintViolationException if the {@link RelationshipLookupSpecific} violates constraints enforced by the persistence service
+     */
+    Optional<RelationshipResponse> findSpecific(
```

Comment:
I don't think we need a Lookup DTO, arguments could do the job as well, especially since all of them are required anyway. Thing and Metadata do it that way, I guess. However, we could discuss that.